client/my-sites/email/email-providers-comparison/stacked/index.tsx

```diff
@@ -86,12 +92,47 @@ const EmailProvidersStackedComparison = ( {
 		)
 	);
 
+	const currentUserCanAddEmail = canCurrentUserAddEmail( domain );
+
+	const ownerUserName = useDomainOwnerUserName( selectedSite, domain );
+
+	const isPrivacyAvailable = domain?.privacyAvailable;
+
+	const contactOwnerUrl = `https://privatewho.is/?s=${ selectedDomainName }`;
+
+	const loginUrl = loginUrlWithUserNameAndRedirectToEmailProvidersComparison(
+		ownerUserName,
+		selectedSite?.slug,
+		selectedDomainName
+	);
+
+	/**
+	 * This function builds the event 'calypso_email_providers_owner_contact_click'
+	 * or 'calypso_email_providers_user_login_click' depending on the event type passed as parameter.
+	 *
+	 * @param { "owner_contact" | "user_login" } eventType
```

Comment:
That annotation seems superfluous given it just duplicates what we already have in the function's signature.

Reply:
As the key is built dynamically, I thought it was good to keep the final keys so other developers would find them easily.

Reply:
Note I was only talking about the `@param` annotation here. Good idea mentioning the Track events in the comment above although I think it's better to avoid building them dynamically in the code (as someone could forget to update that comment 🙂).